src/ripple/app/main/DBInit.h

```diff
@@ -63,15 +73,14 @@ inline constexpr auto TxDBName{"transaction.db"};
 
 inline constexpr
 #if (ULONG_MAX > UINT_MAX) && !defined(NO_SQLITE_MMAP)
-    std::array<char const*, 6>
+    std::array<char const*, 4>
         TxDBPragma
 {
     {
 #else
-    std::array<char const*, 5> TxDBPragma {{
+    std::array<char const*, 3> TxDBPragma {{
```

Comment:
The preprocessor guards make this declaration hard to read. Now that we're on c++-17 I think we can simplify this to:
```

inline constexpr std::array TxDBPragma
{
    "PRAGMA page_size=4096;", "PRAGMA journal_size_limit=1582080;",
        "PRAGMA max_page_count=2147483646;",
#if (ULONG_MAX > UINT_MAX) && !defined(NO_SQLITE_MMAP)
        "PRAGMA mmap_size=17179869184;",
#endif
};
```

Reply:
Nice. Done.